crates/bdk/Cargo.toml

```diff
@@ -14,7 +14,7 @@ rust-version = "1.63"
 
 [dependencies]
 miniscript = { version = "10.0.0", features = ["serde"], default-features = false }
-bitcoin = { version = "0.30.0", features = ["serde", "base64", "rand-std"], default-features = false }
+bitcoin = { version = "0.30.0", features = ["serde", "base64"], default-features = false }
```

Comment:
we should also be removing the `target = "wasm32"` dependencies (or making them dev dependencies to get tests to pass).